lib/const.rb

```diff
@@ -298,6 +298,7 @@
 PY_SETUP_INSTALL_OPTIONS         ||= "#{PY_SETUP_INSTALL_OPTIONS_NO_SVEM} --single-version-externally-managed"
 PY3_BUILD_OPTIONS                ||= '--wheel --no-isolation'
 PY3_INSTALLER_OPTIONS            ||= "--destdir=#{CREW_DEST_DIR} --compile-bytecode 2 dist/*.whl"
+PY3_PIP_RETRIES                  ||= 5
```

Comment:
Still hardcoded?  How would you override the value?

Reply:
Ah good point!

Reply:
Hopefully this change works better...